### PR TITLE
GRAD2-174: the initial implementation is committed for the finalized …

### PR DESCRIPTION
**Pull request by infstar**
…document of ongoing updates from TRAX to GRAD.

GRAD2-174: the initial implementation is committed for the finalized document of ongoing updates from TRAX to GRAD.

### CODE REVIEW

File: api/src/main/java/ca/bc/gov/educ/api/trax/model/dto/ConvGradStudent.java [hunk 1]
```diff
@@ -14,33 +14,33 @@
 @AllArgsConstructor
```
Comment (cditcher) [inline api/src/main/java/ca/bc/gov/educ/api/trax/model/dto/ConvGradStudent.java:14]: Looks like only spacing changes on this file. No need to commit.

File: api/pom.xml [hunk 2]
```diff
@@ -18,6 +18,8 @@
             src/main/java/ca/bc/gov/educ/api/trax/config/**,
             src/main/java/ca/bc/gov/educ/api/trax/constant/**,
             src/main/java/ca/bc/gov/educ/api/trax/choreographer/**,
+            src/main/java/ca/bc/gov/educ/api/trax/messaging/**,
```
Comment (cditcher) [inline api/pom.xml:21]: I can understand removing scheduler from sonar, but why messaging?

**Review by cditcher — CHANGES_REQUESTED** [review]

File: api/pom.xml (hunk 2 above)
Comment (infstar) [reply api/pom.xml:21]: All other NATS feature enabled APIs get the messaging package put to sonar exclusion with the following reason.
Any changes made in it like a new topic is added or some NATS configuration is changed will cause the coverage issue on the new code even though the message package is in junit exclusion.

File: api/src/main/java/ca/bc/gov/educ/api/trax/model/dto/ConvGradStudent.java (hunk 1 above)
Comment (infstar) [reply api/src/main/java/ca/bc/gov/educ/api/trax/model/dto/ConvGradStudent.java:14]: Yes, it is not required.  Do you want another commit into this PR to revert it?

Comment (cditcher) [reply api/src/main/java/ca/bc/gov/educ/api/trax/model/dto/ConvGradStudent.java:14]: Not necessary.

**Review by cditcher — APPROVED** [review]

**Pull request merged**